thavalon-webapp/src/components/formComponents/InputElement.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import "../../styles/formStyles/InputElement.scss";
+
+export interface InputElementProps {
+    "type": string,
+    "label": string,
+    "name": string,
+    "required": boolean,
+    "minLength"?: number,
+    "formRef": any, // not fully clear what the type of register from react-hook-form is
```

Comment:
I did a bit of research, and I _think_ it's a `React.RefObject<HTMLInputElement>`

Reply:
Trying that gives:

Property 'current' is missing in type '{ <TFieldElement extends FieldElement<LoginData>>(rules?: Partial<{ required: string | boolean | ValidationValueMessage<boolean>; min: ValidationRule<Key>; ... 4 more ...; validate: Validate | Record<...>; }> | undefined): (ref: (TFieldElement & HTMLInputElement) | ... 3 more ... | null) => void; (name: "email" | "p...' but required in type 'RefObject<HTMLInputElement>'.ts(2741)